README.md

```diff
@@ -1,4 +1,35 @@
-# iroha2-block-explorer-backend
+# Iroha 2 Blockchain Explorer Backend
+
+This readme provides an overview of the Iroha 2 blockchain explorer backend and instructions on how to install, run, and utilize the explorer's backend features.
+
+## Installation
+
+To set up the Iroha 2 blockchain explorer backend, follow these steps:
+
+1. Install Rust.
```

Comment:
any requirements on the version of Rust?

Reply:
No, It works with the latest version. But I think it would be better to document that it works with ```rust 1.70.0```

Reply:
It works for the latest version of Rust ```1.7.0``` . But It would be better to mention.